src/TimeService.Data.Provider/IDataProvider.cs

```diff
@@ -1,9 +1,12 @@
-﻿using LT.DigitalOffice.Kernel.Database;
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Kernel.Database;
+using LT.DigitalOffice.Kernel.Enums;
 using LT.DigitalOffice.TimeService.Models.Db;
 using Microsoft.EntityFrameworkCore;
 
 namespace LT.DigitalOffice.TimeService.Data.Provider
 {
+    [AutoInject(InjectType.Scoped)]
```

Comment:
That was Transient before, are you sure that Scoped is better?

Reply:
Yes